src/main/java/dk/cs/aau/huppaal/controllers/HUPPAALController.java

```diff
@@ -403,6 +387,27 @@ private void initializeMenuBar() {
             HUPPAAL.save();
         });
 
+        menuBarPreferencesUppaalLocation.setOnAction(event -> {
+            // Dialog title
+            final FileChooser filePicker = new FileChooser();
+            filePicker.setTitle("Choose UPPAAL server file");
+            filePicker.getExtensionFilters().add(new FileChooser.ExtensionFilter("UPPAAL server", "*.exe"));
```

Comment:
on macOs/linux the server will not include *.exe

Reply:
I have removed the filter, I'm unsure whether the file type should be handled in any way or just caught by exceptions when trying to run it